build.gradle

```diff
@@ -2151,7 +2151,7 @@ project(':streams') {
 
   task genStreamsConfigDocs(type: JavaExec) {
     classpath = sourceSets.main.runtimeClasspath
-    mainClass = 'org.apache.kafka.streams.StreamsConfig'
+    mainClass = 'org.apache.kafka.streams.internals.InternalStreamsConfig'
```

Comment:
We use `main` to generate the config HTML for the webpage, so update to use internal one going forward